### ISSUE TITLE
learning rate for prelong 1000 epochs

### ISSUE THREAD

**Issue by verigle** [JiaRenChang/PSMNet#62]
hi
In your paper , you prelong train for 1000 epochs for submit on kitti,
is the 200 epochs for 0.001 , and last 800 epochs  for 0.0001?

**Comment by JiaRenChang** [comment]
@verigle 
No, we keep same proportion (2:1) in long learning rate scheduling.

**Comment by verigle** [comment]
thank you!

**Comment by auxlchen** [comment]
> @verigle
> No, we keep same proportion (2:1) in long learning rate scheduling.

I don't really understand the scheduling. Could you specify this?
